.github/workflows/triage-labels.yml

```diff
@@ -27,7 +27,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: initial labeling
-        uses: andymckay/labeler@master
+        uses: actions/github-script@v6
         with:
-          add-labels: "triage"
-          remove-labels: "awaiting_response"
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.issues.addLabels({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                labels: ['triage']
+              })
+
+            github.rest.issues.removeLabel({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                name: 'awaiting_response'
+              })
```

Comment:
Turns out this is pretty straightforward.
The github-script repo has an [example for adding a label](https://github.com/actions/github-script#apply-a-label-to-an-issue)

Here are the docs for [removeLabel](https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/main/docs/issues/removeLabel.md) and [addLabels](https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/main/docs/issues/addLabels.md)

Reply:
I just found the [swap-labels](https://github.com/dbt-labs/actions/blob/main/.github/workflows/swap-labels.yml) action and will switch to that

Reply:
We do it across 5 repos, that's why I centralized it.